vars.tf

```diff
@@ -0,0 +1,86 @@
+variable "project" {
+  type        = string
+  description = "Three letter project key"
+}
+
+variable "stage" {
+  type        = string
+  description = "Stage for this ip"
+}
+
+variable "location" {
+  type        = string
+  description = "Azure location to use for the backup"
+}
+
+variable "subscription_id" {
+  type        = string
+  description = "The Subscription ID to use"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "Azure Resource Group to use"
+}
+
+variable "backup_application_id" {
+  type        = string
+  description = "Application ID used for backup"
+}
+
+variable "backup_application_secret" {
```

Comment:
This shouldn't also be named application secret, since it is a secret of a service principal